### PR TITLE
Made isFullscreen() work on Android

### PR DESCRIPTION
**Pull request by GrantOjanen**
Tested on Android 8 and Android 4.4. I haven't tested it with live wallpapers or daydreams, but it should always return true on those. It only returns false when in multi-window mode in activities or fragments.

Also I made it so FullscreenTest now displays true/false for fullscreen.

### CODE REVIEW

**Comment by intrigus** [comment]
Isn't "android fullscreen" quite different from desktop/web fullscreen?

**Comment by GrantOjanen** [comment]
@intrigus What do you mean by "different"? Android 7 has a split window mode as well as free-form window mode. The free-form window mode must be activated on tablets/phones. I've never actually used an Android 7+ laptop. I don't think developers can force their activity to windowed/fullscreen mode.

File: backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidApplicationBase.java [hunk 1]
```diff
@@ -91,4 +88,8 @@
 	/** Returns the Handler object created by the application
 	 * @return The {@link Handler} object created by the application */
 	Handler getHandler ();
+
+	/** Returns the current Android Activity
+	 * @return The current Android {@link Activity} */
+	Activity getActivity ();
```
Comment (obigu) [inline backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidApplicationBase.java]: You probably don't need this, use `getContext()` and check if it returns instance of `Activity`

File: backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidGraphics.java [hunk 2]
```diff
@@ -718,10 +716,10 @@ public void requestRendering () {
 
 	@Override
 	public boolean isFullscreen () {
-		return true;
+		Activity activity = app.getActivity();
+		return activity == null || Build.VERSION.SDK_INT < 24 || !activity.isInMultiWindowMode();
```
Comment (obigu) [inline backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidGraphics.java]: Not very readable in my opinion but I accept it's a matter of personal taste

File: backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidApplicationBase.java (hunk 1 above)
Comment (noblemaster) [reply backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidApplicationBase.java]: I don't think ``Context`` supports ``isInMultiWindowMode()``?

File: backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidGraphics.java (hunk 2 above)
Comment (noblemaster) [reply backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidGraphics.java]: I would probably put brackets ( ... ) around the sub-conditions, but maybe I am little bit strict there too :)

**Comment by noblemaster** [comment]
Let's wait for the checks to complete (still pending) and then merge.

File: backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidApplicationBase.java (hunk 1 above)
Comment (obigu) [reply backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidApplicationBase.java]: `Activity` is a subclass of `Context` and in standard Android apps getContext() returns an Activity being LiveWallpapers an exception. I'm suggesting to check if `getContext()`returns an instance of `Activity` to then be able to call `isInMultiWindowMode()`.

Comment (noblemaster) [reply backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidApplicationBase.java]: I see your point, there indeed is a ``getContext()`` method. I agree with obigu, let's remove the ``getActivity()`` and use the ``getContext()``! :)

**Comment by MrStahlfelge** [comment]
It is a good addition in my opinion, but has conflicts and unresolved requests for changes.
Please reopen the PR if you solved that.

**Pull request closed without merging**